mlflow/server/handlers.py

```diff
@@ -126,6 +126,7 @@ def _create_experiment():
     request_message = _get_request_message(CreateExperiment())
     experiment_id = _get_store().create_experiment(request_message.name,
                                                    request_message.artifact_location)
+    experiment_id = str(experiment_id)  # local stores use integer instead of str
```

Comment:
updated this line to keep the local_store_experiment_id vs experiment_id convention throughout handlers.py